test/CMakeLists.txt

```diff
@@ -45,4 +48,4 @@ if(UNIX AND NOT APPLE)
             LINK_FLAGS "-Wl,--disable-new-dtags")
 endif()
 
-add_test(NAME menoh_test COMMAND menoh_test)
+#add_test(NAME menoh_test COMMAND menoh_test)
```

Comment:
Why `enable_testing()` is removed and `add_test()` is commented out?

Reply:
It didn't worked before. I fixed it by new commit. Please check.

Reply:
Thanks. I checked that `ctest` works.

Reply:
Thanks! Do you have any other review comments?